modules/geotiff/src/lib/load-geotiff.ts

```diff
@@ -19,31 +18,33 @@ interface TiffOptions {
 /**
  * Opens an OME-TIFF via URL and returns data source and associated metadata for first image.
  *
- * @param {(string | File)} source url or File object.
+ * @param {(string | Blob | GeoTIFF)} source url or File object.
```

Comment:
Should we have types both in the jsdoc and in the function itself? Seems to me it would be preferable to only manage types in a single place.

Reply:
These are residual from our documentation being generated from JSDoc and not TS. It's preferable to me to only have TS.